exotic.py

```diff
@@ -235,7 +235,7 @@ def new_scrape(filename="eaConf.json"):
         "select"   : "pl_name,hostname,tran_flag,pl_massj,pl_radj,pl_ratdor,"
                      "pl_orbper,pl_orbpererr1,pl_orbpererr2,pl_orbeccen,"
                      "pl_orbincl,pl_orblper,pl_tranmid,pl_tranmiderr1,pl_tranmiderr2,"
-                     "st_teff,st_met,st_logg,st_mass,st_rad,ra,dec",
+                     "st_teff,st_tefferr1,st_met,st_meterr1,st_logg,st_loggerr1,st_mass,st_rad,ra,dec",
```

Comment:
After consulting with Kyle, it seems that scraping both parameters is not needed for limb darkening. If we scraped both the + and - uncertainties, it would cause a longer load time for users to have the eaConf.json on their computer (and, depending on the paper, they're usually the same). It is possible to do of course and can be implemented if needed. I'm not sure what error `3` is, but `2` is the - and `1` is the + for uncertainties. If needed for the LDTK, I could implement both `1` and `2` to find the larger of the two for limb darkening calculations.

Reply:
I would implement both `1` and `2` and then take the geometric mean, a la EXCALIBUR.

Reply:
Taking the geometric mean is being done in my separate environment and will be added to EXOTIC's LD calculations function.

Reply:
Ok great thanks.